### ISSUE TITLE
Dockerize example

### ISSUE THREAD

**Issue by S0c5** [AdrianZelada/microservicesNode#1]
## Description

In order to setup quickly the app, its helpfully dockerize the app.

**Comment by AdrianZelada** [comment]
Yes Sr.... I wrote a post about microservices “Microservicios Ejemplo de Flujo” https://medium.com/mycodebad/microservicios-ejemplo-de-flujo-f45720a9b278
